src/Core/Impl/Extensions/TaskExtensions.cs

```diff
@@ -22,9 +22,9 @@
 namespace Microsoft.Python.Core {
     public static class TaskExtensions {
         public static void SetCompletionResultTo<T>(this Task<T> task, TaskCompletionSourceEx<T> tcs) 
-            => task.ContinueWith(SetCompletionResultToContinuation, tcs, CancellationToken.None, TaskContinuationOptions.ExecuteSynchronously, TaskScheduler.Default);
+            => task.ContinueWith(SetCompletionResultToContinuationEx, tcs, CancellationToken.None, TaskContinuationOptions.ExecuteSynchronously, TaskScheduler.Default);
         
-        private static void SetCompletionResultToContinuation<T>(Task<T> task, object state) {
+        private static void SetCompletionResultToContinuationEx<T>(Task<T> task, object state) {
```

Comment:
`SetCompletionResultToContinuationEx` is only used in `SetCompletionResultTo`, which itself is unused. Do you need this code anymore if you're not using `TaskCompletionSourceEx`?

Reply:
I deleted the unused code